context/selection-of-signal-for-I.md

```diff
@@ -0,0 +1,165 @@
+        * Market based reporting doesn’t incentivize making applications more energy efficient.
+
+
+## References
+
+_List here any references to external sources, plus any specific insight from the reference which would aid in this discussion topic._
+
+* SRMER are very good at assessing the short-term impact of an intervention
+    * Source: [How Well Do Emission Factors Approximate Emission Changes from Electricity System Models?](https://pubs.acs.org/doi/abs/10.1021/acs.est.2c02344)
+    * Twitter Discussion about it: [Twitter Thread from Eric Hittinger](https://twitter.com/ElephantEating/status/1590741977880211458)
+    * Twitter Discussion providing a contrary view: [Twitter Thread from AukeHoekstra responding to the Twitter Thread by Eric Hittinger](https://twitter.com/AukeHoekstra/status/1620525739803291648) With clarification for load shifting: [https://twitter.com/Helmut_Lehmeyer/status/1620766545793404928](https://twitter.com/Helmut_Lehmeyer/status/1620766545793404928)
+* LRMER are very strong predictors of long-term impacts of interventions. AER are a strong proxy for LRMER, while SRMER does not capture long-term impacts of interventions
+    * [Planning for the evolution of the electric grid with a long-run marginal emission rate](https://www.cell.com/iscience/fulltext/S2589-0042(22)00185-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2589004222001857%3Fshowall%3Dtrue)
+    * [Short-run marginal emission rates omit important impacts of electric-sector interventions](https://www.pnas.org/doi/10.1073/pnas.2211624119)
+* Marginal emission factors are based on estimation models and their value cannot be verified. This is because there is no “ground truth” on the true value as the specific marginal generator cannot be verified. But there is good academic research showing that marginal emissions can be calculated and are a meaningful signal.
+    * [The US RTO PJM states that “marginal units – and the marginal emissions rates based on them –, cannot provide any prediction of the results of an action. ](https://pjm.com/-/media/etools/data-miner-2/marginal-emissions-primer.ashx)
+    * [“All marginal emissions rate estimation methodologies are imperfect for one](https://media.rff.org/documents/Report_22-08.pdf)
+    * [reason or another”](https://media.rff.org/documents/Report_22-08.pdf)
+    * [“There is tremendous variation in the way that MEFs can be estimated. Depending on the specific methodology, data sources, and assumptions used, no one estimate is likely to match another. Even two approaches using the same methodology could use different input data or assumptions to estimate marginal emissions impact, leading to two different and sometimes inconsistent estimates.This means that there is no single source of “truth” for marginal emissions impact.”](https://cebi.org/wp-content/uploads/2022/11/Guide-to-Sourcing-Marginal-Emissions-Factor-Data.pdf)

```

Comment:
I think this overstates the uncertainly around marginal emissions. There's considerable consensus in the academic literature that marginal emissions are the correct metric for measuring the impact of interventions.